Flutter/assistview/getting-started.md

```diff
@@ -60,30 +60,132 @@ Add an assist widget with the required property, [messages].
 
```

Comment:
Initialize assist widget - update proper widget name


AI AssistView - all place you should use this casing